### PR TITLE
Added dynamic fee test: common logics for writing tests (WIP)

### PR DESCRIPTION
**Pull request by aeharvlee**
## Proposed changes

Klaytn will support Dynamic gas price policy, so we should ensure that caver-java will work fine in dynamic gas price policy.

Test case reference: https://github.com/jimni1222/caver-js/blob/4dc316da29297607f955493f5ad68e7625bf5ebc/test/dynamicBaseFee.js

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: core/src/test/java/com/klaytn/caver/DynamicFeeTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                        .setGas(BigInteger.valueOf(6000000))
+        );
+        caver.wallet.sign(rich.getAddress(), valueTransfer);
+        caver.rpc.klay.sendRawTransaction(valueTransfer).send();
+    }
+
+    private void generateTxsBomb(int num) throws IOException {
+        if (num == 0) {
+            num = 100;
+        }
```
Comment (jimni1222) [inline core/src/test/java/com/klaytn/caver/DynamicFeeTest.java:50]:
```suggestion
    private void generateTxsBomb() throws IOException {
        generateTxsBomb(100)
    }
    private void generateTxsBomb(int num) throws IOException {
```
How about changing like this?

File: core/src/test/java/com/klaytn/caver/DynamicFeeTest.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                    TxPropertyBuilder.smartContractDeploy()
+                            .setFrom(sender.getAddress())
+                            .setGas(BigInteger.valueOf(5000000))
+                            .setInput(input)
+                            .setNonce(senderNonce)
+            );
+            senderNonce = senderNonce.add(BigInteger.valueOf(1));
+            caver.wallet.sign(sender.getAddress(), smartContractDeploy);
+            String rawTx = smartContractDeploy.getRLPEncoding();
+            batchRequest.add(caver.rpc.klay.sendRawTransaction(rawTx));
```
Comment (jimni1222) [inline core/src/test/java/com/klaytn/caver/DynamicFeeTest.java:65]:
```suggestion
            caver.wallet.sign(sender.getAddress(), smartContractDeploy);
            String rawTx = smartContractDeploy.getRLPEncoding();
            batchRequest.add(caver.rpc.klay.sendRawTransaction(rawTx));
            senderNonce = senderNonce.add(BigInteger.valueOf(1));
```

Comment (jimni1222) [reply core/src/test/java/com/klaytn/caver/DynamicFeeTest.java:65]: Just change the sequence

File: core/src/test/java/com/klaytn/caver/DynamicFeeTest.java (hunk 1 above)
Comment (aeharvlee) [reply core/src/test/java/com/klaytn/caver/DynamicFeeTest.java]: Thanks for your feedback :) 
I added it!

**Review by jimni1222 — APPROVED** [review]

**Pull request merged**